UnityProject/Assets/Scripts/Managers/LobbyManager/LobbyManager.cs

```diff
@@ -321,6 +353,16 @@ private void DetermineUIScale()
 			}
 		}
 
+		private async void LoginTimer()
+		{
+			await Task.Delay(14000);
+			if (cancelTimer) return;
+			lobbyDialogue.ShowLoadingPanel("This is taking longer than it should..\n\n If it continues, try disabling your VPNs and installing the game in full English path.");
```

Comment:
loadingPanelScript.SetActive(true);
Unity will complain about not being on the main game thied

Reply:
but everything else is also working on async?

Reply:
Is it?

Reply:
Yeah but it hasn't hit an awite yet so sill on mine thied

Reply:
I've tested this multiple times and there doesn't seem to be any issues.
Also we're calling `lobbyDialogue.ShowAlphaPanel();` and `lobbyDialogue.ShowLoadingPanel();` without awaiting so clearly its working just fine.

Reply:
the point I'm making is that because it hasn't hit an await yet it still on the main thread, but anything after a await  is on a non main thread

Reply:
why is it not complaining then?

Reply:
is it perhaps not complaining because we're not accessing the `gameObject` property and have a direct reference to what we want to work with?